tutorials/plugins/editor/making_plugins.rst

```diff
@@ -161,7 +161,7 @@ clicked. For that, we'll need a script that extends from
 
 
     func _enter_tree():
-        connect("pressed", self, "clicked")
+        connect("pressed", clicked)
```

Comment:
That works, but it's actually the old style signal connection. The new syntax (and preferred for tutorials) should be:
```suggestion
        pressed.connect(clicked)
```